exchange/c2cx/api.go

```diff
@@ -481,6 +482,34 @@ func (c *Client) MarketSell(symbol TradePair, amount decimal.Decimal, customerID
 	return orderID, nil
 }
 
+type GetTickerResponse struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+	Data    struct {
+		Timestamp string  `json:"timestamp,omitempty"`
+		High      float64 `json:"high,omitempty"`
+		Last      float64 `json:"last,omitempty"`
+		Low       float64 `json:"low,omitempty"`
+		Buy       float64 `json:"buy,omitempty"`
+		Sell      float64 `json:"sell,omitempty"`
+		Volume    float64 `json:"volume,omitempty"`
+	} `json:"data"`
+}
+
+func (c *Client) GetTicker(symbol TradePair) (*GetTickerResponse, error) {
+	params := url.Values{}
+	params.Add("symbol", string(symbol))
+	data, err := c.get(getTicker, params)
+	if err != nil {
+		return nil, err
+	}
+	var getTickerResponse GetTickerResponse
+	if err = json.Unmarshal(data, &getTickerResponse); err != nil {
+		return nil, NewOtherError(err)
+	}
+	return &getTickerResponse, err
```

Comment:
check getTickerResponse.Code for error before returning (see other methods for example)